UberSignals.podspec

```diff
@@ -9,10 +9,13 @@ Pod::Spec.new do |s|
   s.source = { :git => 'https://github.com/uber/signals-ios.git', :tag => s.version }
   s.requires_arc = true
 
-  s.ios.deployment_target = '7.0'
+  s.ios.deployment_target = '8.0'
```

Comment:
The framework's deployment target is `8.0` and not in sync with the `podspec` file. That is why I decided to update it. As an option, we can change the project setting instead.